examples/dataset/object_detection_2d/pyproject.toml

```diff
@@ -7,7 +7,8 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-kolena = {extras = ["metrics"], version = "^1.4.0"}
+# TODO: use official release
+kolena = {extras = ["metrics"], path = "../../..", develop = true}
```

Comment:
This is not necessary right?

Reply:
Stashed these changes for now. I will update the example after today's release, assuming this PR is merged.